pkg/auth/arn_test.go

```diff
@@ -82,8 +82,7 @@ func TestArnMatch(t *testing.T) {
 		{"arn:lakefs:repos::b:myrepo", "arn:lakefs:repos::b:myrepo", true},
 		{"arn:lakefs:repos::b:*", "arn:lakefs:repos::b:myrepo", true},
 		{"arn:lakefs:repos::b:myrepo", "arn:lakefs:repos::b:*", false},
-		{"arn:lakefs:repos::b/myrepo", "arn:lakefs:repos::b/*", false},
-		{"arn:lakefs:repos::b/*", "arn:lakefs:repos::b/myrepo", true},
```

Comment:
Can you fix these examples instead of deleting them?

Reply:
Sorry, but again no: these test the previous behaviour of allowing a `/` to separate account-ID from resource-ID.  This is simply no longer allowed.